Prac07/guitarTest.py

```diff
@@ -0,0 +1,31 @@
+from Prac07.guitar import Guitar
+
+
+def main():
+
+    i = 0
+    guitars = []
+    while True:
+        name = input("Name: ")
+        if not name:
+            break
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitar_object = Guitar(name, year, cost)
+        guitars.append(guitar_object)
+        print(guitar_object)
+        i += 1
+
+    print("These are my guitars:")
+    for guitar_number in range(0,len(guitars)):
+        if guitars[guitar_number].is_vintage():
+            vintage_string = "(vintage)"
+        else:
+            vintage_string = ""
+
+        print("Guitar {}: {:>20} ({}), worth ${:10,.2f} {}".format(guitar_number + 1,
```

Comment:
I don't know witch way is better but it is possible to setup the output in the class, (allows for re-usability);

guitar.py
"""    def __str__(self):
           return "{:>20} ({}), worth ${:10,.2f}".format(self.name, self.year, self.cost)"""